lib/index.test.js

```diff
@@ -399,7 +461,7 @@ describe('lib/index', () => {
     await expect(verifyDeps({ autoUpgrade: true, dir, logger })).rejects.toEqual(error);
   });
 
-  test('autoUpgrade modules', async () => {
+  test.skip('autoUpgrade modules', async () => {
```

Comment:
Also here, just making sure the skip is staying!

Reply:
this is intentional, this tests need to be reviewed